cli/cmd/lql.go

```diff
@@ -304,11 +327,59 @@ func runQuery(cmd *cobra.Command, args []string) error {
 		err = queryErrorCrumbs(query, err)
 		return errors.Wrap(err, msg)
 	}
-	if data, ok := response["data"]; ok {
-		return cli.OutputJSON(data)
+	// output
+	if err = cli.OutputJSON(response.Data); err != nil {
+		return err
+	}
+	// fail_on_count post
+	if lqlCmdState.FailOnCount != "" {
+		isFail, err := co.isFail(len(response.Data))
+		if err != nil {
+			return err
+		}
+		if isFail {
+			os.Exit(9)
+		}
 	}
-	if err := cli.OutputJSON(response); err != nil {
-		return errors.Wrap(err, "unable to format json response")
+	return nil
+}
+
+const operationRE = `^(>|>=|<|<=|={1,2}|!=)\s*(\d+)$`
+
+type countOperation struct {
+	operator string
+	num      int
+}
+
+func (co *countOperation) parse(s string) error {
+	re := regexp.MustCompile(operationRE)
+
+	s = strings.TrimSpace(s)
+
+	var op_parts []string
+	if op_parts = re.FindStringSubmatch(s); s == "" || op_parts == nil {
+		return errors.New(
+			fmt.Sprintf("count operation (%s) is invalid", s))
 	}
+	co.num, _ = strconv.Atoi(op_parts[2])
```

Comment:
always check errors 🙏🏽

Reply:
I tend to agree with you.  But it seems unnecessary to check that `\d+` regex can be converted from string to int; no?

Reply:
Super valid point!  I withdraw my comment 🤝